requirements_dev.txt

```diff
@@ -1,4 +1,7 @@
 -r requirements.txt
 bumpversion==0.5.3
 wheel==0.29.0
+django_extensions==1.7.6
+django-braintree==1.3.4
+django-stripe-payments==2.0.0
```

Comment:
`django-braintree` and `django-stripe-payments` not necessary - just the client libraries:
https://github.com/braintree/braintree_python
https://github.com/stripe/stripe-python

Ideally, we shouldnt require them unless they are being used, although this can be handled in another issue